### PR TITLE
tldr-lint, tldrl: add Polish translations

### PR DESCRIPTION
**Pull request by chanibal**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pl/common/tldrl.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# tldrl
+
+> This command is an alias of `tldr-lint`.
```
Comment (navarroaxel) [inline pages.pl/common/tldrl.md:3]: Did you forget this page?
```suggestion
> Te polecenie jest aliasem `tldr-lint`.
```

File: pages.pl/common/tldrl.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://github.com/tldr-pages/tldr-lint>.
+
+- View documentation for the original command:
```
Comment (navarroaxel) [inline pages.pl/common/tldrl.md:6]:
```suggestion
- Zobacz dokumentację orginalnego polecenia:
```

File: pages.pl/common/tldrl.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# tldrl
+
+> This command is an alias of `tldr-lint`.
+> More information: <https://github.com/tldr-pages/tldr-lint>.
```
Comment (navarroaxel) [inline pages.pl/common/tldrl.md:4]:
```suggestion
> Więcej informacji: <https://github.com/tldr-pages/tldr-lint>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.pl/common/tldrl.md (hunk 1 above)
Comment (chanibal) [reply pages.pl/common/tldrl.md:3]: Yes, it looks like I translated this but broke the commit and threw in the untranslated one.  
Fixing this now, sorry.

Btw, it should be "To polecenie", not "Te polecenie".

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @chanibal

**Pull request merged**